org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/contentassist/CompletionProposalReplacementProvider.java

```diff
@@ -246,7 +246,8 @@ private String getTextEditText(CompletionProposal proposal, CompletionItem item,
 				appendLambdaExpressionReplacement(completionBuffer, proposal);
 				break;
 			default:
-				appendReplacementString(completionBuffer, proposal);
+				boolean overloadedMethodProposal = item.getLabelDetails() == null ? false : "(...)".equals(item.getLabelDetails().getDescription());
```

Comment:
I see, as an example :

```
labelDetails = CompletionItemLabelDetails [
    detail = "(...)"
    description = "3 overloads"
  ]
```

Try with this to confirm it works (and collapsed completion items enabled) :

```java
package test1;

public class Main {
    public void doStuff() {
        Exam ex = new Exam();
        ex.getQuest|
    }
}

```

```java
package test1;

public class Test {

    public String[] getQuestions(int number) {
        return null;
    }

    public String[] getQuestions(boolean multipleChoice) {
        return null;
    }
}
```

```java
package test1;

public class Exam extends Test {

    public String[] getQuestions() {
        return null;
    }
}
```